python/ray/train/predictor.py

```diff
@@ -179,21 +192,21 @@ def _predict_pandas(self, data: "pd.DataFrame", **kwargs) -> "pd.DataFrame":
         raise NotImplementedError
 
     @DeveloperAPI
-    def _predict_arrow(self, data: "pyarrow.Table", **kwargs) -> "pyarrow.Table":
-        """Perform inference on an Arrow Table.
+    def _predict_numpy(
```

Comment:
can we update the docstring of Predictor class to reference _predict_numpy instead of_predict_arrow?

Also need to update the docs here https://docs.ray.io/en/latest/ray-air/predictors.html#developer-guide-implementing-your-own-predictor